vars/writeBuildInfo.groovy

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/groovy
+
+import groovy.json.JsonBuilder
+import groovy.json.JsonSlurperClassic
+import org.feedhenry.Utils
+
+def call(name, versionTxt) {
+    def utils = new Utils()
+    def buildInfoFileName = utils.getBuildInfoFileName()
+    def buildInfo = [:]
+
+    if(fileExists(buildInfoFileName)) {
```

Comment:
Yes, unfortunately the whole reason i ended up doing this build-info.json thing was because of fh-messaging, previously we had just been using the VERSION.txt.